frontend/src/views/camp/Collaborators.vue

```diff
@@ -39,67 +39,27 @@ Displays collaborators of a single camp.
         :title="$tc('views.camp.collaborators.inactiveCollaborators')"
       >
         <v-list>
-          <inactive-collaborator-list-item
+          <collaborator-list-item
             v-for="collaborator in inactiveCollaborators"
             :key="collaborator._meta.self"
             :collaborator="collaborator"
             :disabled="!isManager"
+            inactive
           />
         </v-list>
       </content-group>
-
-      <content-group v-if="isManager" :title="$tc('views.camp.collaborators.invite')">
-        <ValidationObserver v-slot="{ handleSubmit }" ref="validation" mode="passive">
-          <v-form ref="form" @submit.prevent="handleSubmit(invite)">
-            <v-container>
-              <v-row align="start">
-                <v-col cols="12" md="9">
-                  <e-text-field
-                    v-model="inviteEmail"
-                    :name="$tc('views.camp.collaborators.email')"
-                    vee-rules="required|email"
-                    :error-messages="inviteEmailMessages"
-                    :error-count="2"
-                    single-line
-                    aria-autocomplete="none"
-                    :placeholder="$tc('views.camp.collaborators.email')"
-                    @keyup="clearMessages"
-                  />
-                </v-col>
-                <v-col sm="9" md="3">
-                  <e-select
-                    v-model="inviteRole"
-                    :items="[
-                      {
-                        key: 'member',
-                        translation: $tc('entity.camp.collaborators.member'),
-                      },
-                      {
-                        key: 'manager',
-                        translation: $tc('entity.camp.collaborators.manager'),
-                      },
-                      {
-                        key: 'guest',
-                        translation: $tc('entity.camp.collaborators.guest'),
-                      },
-                    ]"
-                    item-value="key"
-                    item-text="translation"
-                    :my="0"
-                    dense
-                    vee-rules="required"
-                  />
-                </v-col>
-                <v-col cols="3">
-                  <button-add type="submit" :loading="isSaving" icon="mdi-account-plus">
-                    {{ $tc('views.camp.collaborators.invite') }}
-                  </button-add>
-                </v-col>
-              </v-row>
-            </v-container>
-          </v-form>
-        </ValidationObserver>
-      </content-group>
+    </v-card-text>
+    <v-divider class="mt-n6" />
+    <v-card-text>
+      <div class="text-right">
+        <dialog-collaborator-create v-if="isManager" :camp="camp()">
+          <template #activator="{ on }">
+            <button-add color="success" icon="mdi-account-plus" v-on="on">
```

Comment:
Have you thought about moving this button up, either to the active collaborators section (mental model "Add a new collaborator") or to the invitations section (mental model "send out a new invitation")?

I feel like this button is one of the most often used features, way more often than the inactive collaborators list.

Reply:
I tried putting it into the toolbar, but it seemed off. I will try to add a button on the left hand side under Mitarbeitende. (because this will always be there, as you have to be a manager).

Reply:
Moved it to the Members and handled it just like on camp admin